src/scope/types.ts

```
@@ -0,0 +1,24 @@
+import { BaseScope, Scope } from "./scope"
+import { Symbol } from "./symbols"
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)